toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -27,6 +31,24 @@
 
 _PAGE_SIZE = 50
 _MAXIMUM_RESULTS_FETCHED = 2000
+_GBQ_TIMEOUT_HTTP_REQUEST = 30  # in seconds
+
+
+class JWTNotValidExcption(GoogleUnauthorized):
```

Comment:
nit: small typo in the name, but WDYT of this instead?
```suggestion
class InvalidToken(GoogleUnauthorized):
```

Reply:
To be more explicit, is it not better to have `InvalidJWTToken`